autorest/plugins/yaml_remover.py

```diff
@@ -0,0 +1,15 @@
+class YamlRemover:
```

Comment:
Can we find a more focused name? CloudErrorPlugin or something?

Reply:
makes sense, switched!